exporter/templates_test.go

```diff
@@ -264,7 +264,6 @@ func TestDeviceAppliedTemplate(t *testing.T) {
 	xrn := xmlRootNodeForQrdaOidWithData(qrdaOid, ei)
 
 	assertXPath(t, xrn, "//entry/procedure", map[string]string{"classCode": "PROC", "moodCode": "EVN"}, nil)
-	assertXPath(t, xrn, "//entry/procedure/templateId[@root='2.16.840.1.113883.10.20.22.4.14']", nil, nil)
```

Comment:
Nothing wrong with this, but was curious why you deleted this line, and the similar one in TestDeviceOrderTemplate(), for this branch.